src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -34,7 +34,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 var explicitImplementation = method.Name.Contains('.');
                 if (!explicitImplementation)
                 {
-                    return "Sealed methods can not be intercepted.";
+                    return "Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.";

```

Comment:
I think I'd rather keep the initial sentence, just replacing "sealed" with "non-virtual".

We *could* add some advice, but I'm not sure how to word it. Maybe something like this:

> Non-virtual methods can not be intercepted. Virtual methods are methods explicitly marked virtual, overriding methods, abstract methods and interface methods.